src/Domain/Creators/Cylinder.hpp

```diff
@@ -143,6 +143,17 @@ class Cylinder : public DomainCreator<3> {
     static size_t lower_bound_on_size() noexcept { return 1; }
   };
 
+  struct DistributionInZ {
+    using type = std::vector<domain::CoordinateMaps::Distribution>;
+    static constexpr Options::String help = {
+        "Select the distribution of grid points along the z-axis in each "
+        "layer. The lowermost layer must have a 'Linear' distribution, because "
```

Comment:
List supported values, i.e. `Linear`, `Logarithmic` and `Inverse` (since `Equiangular` is not a valid choice). Check the distribution is valid in the constructor.

(Have you tried selecting `Equiangular` in z-direction? I'm curious what happens :) )